test/index.test.js

```diff
@@ -191,6 +191,33 @@ describe('Quantcast', function() {
           });
         });
 
+        it('should prefix with _fp.event for multiple labels', function() {
+          analytics.page('Page Name', {}, { Quantcast: { labels: ['customLabel1', 'customLabel2'] } });
+          analytics.called(window._qevents.push, {
+            event: 'refresh',
+            labels: '_fp.event.Page Name,_fp.event.customLabel1,_fp.event.customLabel2',
+            qacct: options.pCode
+          });
+        });
+
+        it('should prefix with _fp.event for multiple labels that have commas in them', function() {
```

Comment:
I'm assuming this is handled elsewhere in the logic?

Reply:
Yeah, was glad to see that. So this test is unrelated to my change.